spec/models/opportunity_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe Opportunity do
+  it { should belong_to(:organization) }
+
+  it { should have_valid(:description).when("Lead an HTML/CSS workshop for
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.